src/App.tsx

```diff
@@ -1,17 +1,28 @@
 import apolloLogo from "./assets/apollo-logo.jpeg";
-
-import "./App.css";
+import styles from "./App.module.css";
+import classnames from "classnames";
 
 function App() {
   return (
-    <>
+    <div className={styles.root}>
+      <a href="https://www.apollo.io/tech-blog" target="_blank">
+        <img
+          src={apolloLogo}
+          className={classnames(styles.logo)}
```

Comment:
In case someone prefers to have multiple classnames, I feel like the classnames package is a well established standard.